modules/openapi-generator/src/main/resources/python-legacy/api.mustache

```diff
@@ -247,8 +247,10 @@ class {{classname}}(object):
         {{/hasProduces}}
         {{#hasConsumes}}
         # HTTP header `Content-Type`
-        header_params['Content-Type'] = self.api_client.select_header_content_type(  # noqa: E501
+        header_list = self.api_client.select_header_content_type(  # noqa: E501
```

Comment:
How about `content_types_list = self....`
header_list sounds like it can contain any kind of headers
I understand that it was historically named that